lib/demo/preview.html.erb

```diff
@@ -1,11 +1,14 @@
+<% choice = 'a' %>
 <div class="demo-assessment">
   <h4 class="question"><%= nl2br question %></h4>
   <ul>
   <% solutions.each do |solution| %>
-  <li class="solution">
+  <div class="solution">
```

Comment:
generally speaking a "solution" for us is a right answer to a problem - when we have multiple solutions they are all right.  Because of the check mark I think these are more answer choices of a multiple choice problem.  There can be several "solutions" that are ways to arrive at the right answer choice.... BUT I say this mostly so you and I are on the same page.  There is nothing stopping a member from calling answer choices solutions in their preview HTML -- as long as they don't put them in their `solutions` field in the posted assessment we are ok.  I will clean this up in http://develop.a15k.org/tech/ terminology table.

Reply:
yeah, the class here really isn't used at all, I'll just remove it to prevent mis-understandings